### PR TITLE
Add op_vault-for-bitcoin-covenants-panel review to bitcoin-review-podcast

### PR DESCRIPTION
**Pull request by satstacker21**
This PR adds [op_vault-for-bitcoin-covenants-panel](https://www.youtube.com/watch?v=LC3lZ9dMRoA) transcript review to the bitcoin-review-podcast directory.

### CODE REVIEW

File: bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md [hunk 1]
```diff
@@ -344,689 +324,641 @@ What I prefer to do is actually use an XPUB that's associated with your super, s
...
 Yeah, exactly.
-Like, you could, yeah, like deposit in the Coinbase and do the bet and then, you know, withdraw, but you know, we saw FTX worked out.
-We saw, you know, the other thousand before them worked out as well.
-Like we need to kind of wait to do this natively.
-And like, I think there is demand like and the shit coin space, like a lot of people are using like, you know, all their trading stuff on there That's like a uni swap and blah blah blah just to like trade stuff quote-unquote trustlessly So there actually is demand for this and it seems like you know I'd rather people use Bitcoin than shit coins to do it So I hope it's
+you could deposit in the Coinbase and do the bet and then, withdraw, but we saw FTX worked out.
+We saw, the other thousand before them worked out as well.
+we need to kind of wait to do this natively.
+And I think there is demand  and the shitcoin space,  a lot of people are using , all their trading stuff on there That's  a uni swap and blah blah blah just to  trade stuff quote-unquote trustlessly 
+So there actually is demand for this and it seems I'd rather people use Bitcoin than shitcoins to do it So I hope...
```
Comment (kouloumos) [inline bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md:808]: This is an example of self-correction
```suggestion
So there actually is demand for this and I'd rather people use Bitcoin than shitcoins to do it.
```

File: bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md [hunk 2]
```diff
@@ -344,689 +324,641 @@ What I prefer to do is actually use an XPUB that's associated with your super, s
...
-Speaker 4: 00:39:30
+James: 00:39:30
 
 I think that's a lot of the work that remains to be done.
 I mean, I think the framework right now gives the end user a lot of configurability.
-And in designing it, I'm trying to avoid any particular configuration that would be like just blatantly unsafe and, you know, like any use that just wouldn't make any sense.
-But at the same time, you know, I mean, things like for your recovery path, are you using a static address there or are you varying it over a descriptor?
-You know, like that does have implications.
+And in designing it, I'm trying to avoid any particular configuration that would be just blatantly unsafe and any use that just wouldn't make any sense.
+But at the same time, I mean, things for your recovery path, are you using a static address there or are you varying it over a descriptor, that does have implications.
```
Comment (kouloumos) [inline bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md:651]:
```suggestion
But at the same time, I mean, things like  for your recovery path, are you using a static address there or are you varying it over a descriptor?
That does have implications.
```

File: bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md [hunk 3]
```diff
@@ -344,689 +324,641 @@ What I prefer to do is actually use an XPUB that's associated with your super, s
...
+And it costs a lot.
+The insurance on this stuff is obnoxious.
+Because this is a very easy to steal asset.
+its best feature is its solubility and, fungibility and transportability.
+So how do you do that cheaper, which everybody cares about, and safer
 And I think this is great for enterprise that has to deal with insurance.
-And soon enough, we're going to see even individuals seeking insurance, right?
-That's what the guys from like Anchor Watch and things like that are trying to do.
+And soon enough, we're going to see even individuals seeking insurance
+That's what the guys from  Anchor Watch and things  that are trying to do.
```
Comment (kouloumos) [inline bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md:687]: example
```suggestion
That's what the guys from AnchorWatch and things like that are trying to do.
```

File: bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md [hunk 4]
```diff
@@ -1515,592 +1416,524 @@ People are still working on covenant proposals that are not announced yet.
...
+But they don't also don't understand that  software doesn't live forever without gardening.
+the stuff that's running on your computer today doesn't run tomorrow.
+And you need the people who understand the code base to sort of upgrade it for new hardware, for new concerns, for, Unix time problems.
 And there's all these things that need to change.
-And you know, when my good friend Steve says, fire the devs, I understand why it feels that way.
-And I kind of agree in a way because, you know, like core, I addressed this in the last episode.
-It kind of feels like this, it's like Internet Explorer has 99% of the market.
+when my good friend Steve says, fire the devs, I understand why it feels that way.
+And I kind of agree in a way because,  core, I addressed this in the last episode.
+It kind of feels  this, it's  Internet Explorer has 99% of the market.
```
Comment (kouloumos) [inline bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md:1506]: You see how removal of "like" sometimes distorts the sentence?

**Review by kouloumos — COMMENTED** [review]
Hey @satstacker21, Thank you for the effort you've put into editing this transcript. Your initiative in utilizing find-and-replace to remove filler words and catch numerous transcription errors from the AI is commendable. It demonstrates a proactive approach to streamlining the review process, which aligns with our goal of finding more productive ways to support our reviewers.

However, it has come to my attention that the application of find-and-replace has inadvertently affected the readability of the transcript in some places, leading to sentences that lack coherence or have altered formatting. While we encourage the exploration of efficient editing techniques, it's crucial to maintain a balance to ensure the transcript remains clear and understandable.

Given the extensive nature of this two-hour transcript, I recognize the significant challenge it poses. We are currently evaluating the feasibility of reviewing such lengthy transcripts on our platform and are keen to hear your insights on this matter.

As it stands, merging the submission in its current state presents a dilemma due to the aforementioned issues. While it's unreasonable to request a complete re-review of such a voluminous transcript, perhaps we can focus on addressing the most critical readability concerns highlighted.

Your feedback on handling lengthy transcripts and any suggestions for improving the review process would be greatly valued. Let's work together to find a solution that upholds the quality of our content without imposing undue burdens on our reviewers.

File: bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md [hunk 5]
```diff
@@ -1,340 +1,320 @@
...
 
 Hey guys.
-Yeah, I'm a Bitcoin developer.
+I'm a Bitcoin developer.
 I spend a lot of my days working on core, but for the last few years, I've been really interested in custody systems and the promise of vaults.
-And last year, I actually implemented a similar vault design on top of OpsyTV, which I'm sure we've covered at some point on the show, maybe.
+And last year, I actually implemented a similar vault design on top of `OP_CTV`, which I'm sure we've covered at some point on the show.
 But I was kind of dissatisfied with those implementations for various reasons, and so decided to come up with this proposal.
-So just want to say that, you know, I mean to make this a lively discussion, you should do your best to rip into this and you won't offend me, you know, kind of regardless of whatever you say, you know, like NVK said, any new proposal for Bitcoin has to be kind of shredded apart and scrutinized.
+So just want to say that to make this a lively discussion, you should do your best to rip into this and you won't offend me, kind of regardless of whatever you say,  NVK said, any new proposal for Bitcoin has to be kind of shredded apart and scrutinized.
```
Comment (kouloumos) [inline bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md:59]: missing like
```suggestion
So just want to say that to make this a lively discussion, you should do your best to rip into this and you won't offend me, kind of regardless of whatever you say, like NVK said, any new proposal for Bitcoin has to be kind of shredded apart and scrutinized.
```

File: bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md [hunk 6]
```diff
@@ -1,340 +1,320 @@
...
-So, okay.
-So, you know, like we have a way now to, to handle the fees.
-Maybe it's, there is an op vault for the op vault.
-Like maybe there's a simpler op vault that you built that just handles fees in case you need more fees.
-Right.
-That's like your fee wallet.
-And the other
+okay.
+we have a way now to handle the fees.
+Maybe it's, there is an `OP_vAULT` for the `OP_vAULT`.
```
Comment (kouloumos) [inline bitcoin-review-podcast/op_vault-for-bitcoin-covenants-panel.md:296]: You have multiple variances of `OP_VAULT` written.

**Review by kouloumos — COMMENTED** [review]
As I said in my first review, it seems you might have utilized a find-and-replace function to remove occurrences of 'like.' However, it's important to note that not every instance of 'like' should be eliminated, as its relevance varies with context.

Because of how you replaced the words, there is a double space at all the places that you did a find-and-replace. Please use that to identify where the "like" should not be removed and add that back to the sentence. You can also spot those by looking at the diff [here](https://github.com/bitcointranscripts/bitcointranscripts/pull/419/files) I gave you only a few of those examples during my review and I was expecting from you to find the rest. As soon as you do that we could merge this.

**Comment by kouloumos** [comment]
Hey @satstacker21, it’s been some time since I last reviewed your submission, and there haven't been any updates since then. Do you have plans to work on the feedback provided?

**Comment by satstacker21** [comment]
I got pretty occupied with family time and my fiat job but I plan to resume
soon.

Regards,

Satstacker

**Comment by kouloumos** [comment]
Hey @satstacker21, are you still planning to work on this?

**Comment by satstacker21** [comment]
No, I am not.

**Pull request closed without merging**